rmc/plotting/expected_vs_significance.R

```diff
@@ -0,0 +1,37 @@
+library(ggplot2)
+
+# Start with the following scenario:
+# O/E = 0 on side A, O/E = 1 on side B
+# Side A and side B have the same number of expected variants
+
+# Calculate chi-square value in the above scenario with a specified
+# expected number of variants in side A (equal to the expected in side B)
+get_chisq_single_break = function(obs1, exp1, obs2, exp2) {
+    return(
+        -2 *
+            (
+                log(dpois(obs1, exp1 * (obs1 + obs2) / (exp1 + exp2))) +
```

Comment:
totally small thing, but this line (and the ones below) could use the log parameter:
```
dpois(obs1, exp1 * (obs1 + obs2) / (exp1 + exp2), log.p = True) +
```